node/_errors.ts

```diff
@@ -83,15 +83,47 @@ export class ERR_INVALID_ARG_TYPE extends NodeTypeError {
   constructor(a1: string, a2: string | string[], a3: unknown) {
     super(
       "ERR_INVALID_ARG_TYPE",
+      // TODO(kt3k): Expression for a2 is wrong.
       `The "${a1}" argument must be of type ${
         typeof a2 === "string"
           ? a2.toLocaleLowerCase()
           : a2.map((x) => x.toLocaleLowerCase()).join(", ")
-      }. Received ${typeof a3} (${a3})`,
+      }.${invalidArgTypeHelper(a3)}`,
     );
   }
 }
 
+export class ERR_INVALID_ARG_VALUE extends NodeTypeError {
```

Comment:
Why `extends NodeTypeError`? Node's is a `TypeError`.

Reply:
I'm not sure about the intention of `NodeTypeError`, but it seems introduced in this PR https://github.com/denoland/deno/pull/7934/files, and it seems being used instead of TypeError in our code base. So I don't think it's good idea to change it for only this one, but rather I suggest we should address them together with others in later PRs.

Reply:
Fair enough. I'll mark this resolved.